File: Code 3 Callouts/Code 3 Callouts/Models/Peds/Witness.vb
Imports Rage

Namespace Models.Peds

    Public Class Witness
        Inherits PedBase

        Public Sub New(ByVal pName As String, ByVal position As Rage.Vector3)
            MyBase.New(pName, PedType.Witness, position)
        End Sub

        Public Sub New(ByVal pName As String, ByVal model As Rage.Model, ByVal position As Rage.Vector3, ByVal heading As Single)
            MyBase.New(pName, PedType.Witness, model, position, heading)
        End Sub

        Protected Friend Sub New(ByVal pName As String, ByVal handle As Rage.PoolHandle)
            MyBase.New(pName, PedType.Witness, handle)
        End Sub

    End Class

End Namespace

File: Code 3 Callouts/Code 3 Callouts/Models/Peds/Witness.cs
using Rage;
using static Stealth.Plugins.Code3Callouts.Common;

namespace Stealth.Plugins.Code3Callouts.Models.Peds
{

	internal class Witness : PedBase
	{

		internal Witness(string pName, Rage.Vector3 position) : base(pName, PedType.Witness, position)
        {
			
		}

		internal Witness(string pName, Rage.Model model, Rage.Vector3 position, float heading) : base(pName, PedType.Witness, model, position, heading)
        {
			
		}

		protected internal Witness(string pName, Rage.PoolHandle handle) : base(pName, PedType.Witness, handle)
        {
			
		}

	}

}